src/ServiceControl/Operations/ErrorImporter.cs

```diff
@@ -81,5 +85,27 @@ protected override Task OnStop(IMessageSession session)
                 return Task.FromResult(0);
             }
         }
+
+        class EnsureCanWriteToForwardingAddress : FeatureStartupTask
```

Comment:
How about making this one non-nested and sharing it between audit and error forwarding?

Reply:
Can do. The only reason I thought it might make sense to keep them separate is that the startup diagnostics will record them by type name and nesting them means it will report Audit+Ensure... and Error+Ensure...